README.md

```diff
@@ -40,12 +40,6 @@ This repo hosts the Dockerfiles used to generate the official images for OSRF pr
 * amd64 [![Docker Build Status](https://img.shields.io/docker/build/osrf/gazebo.svg)](https://hub.docker.com/r/osrf/gazebo/builds/)
 
 
-### [SROS ![Docker Pulls](https://img.shields.io/docker/pulls/osrf/sros.svg) ![Docker Stars](https://img.shields.io/docker/stars/osrf/sros.svg)](https://registry.hub.docker.com/u/osrf/sros/)
```

Comment:
Is removing these badges from the README accidental? It seems like an unrelated change.

Reply:
It was intentional. With official Ros 2 docker images offering security features, I see little need to distract new users from this with older unfinished security approaches.